### PR TITLE
ICP-5919 Make temperature report less frequently than IAS so that ope…

### PR DESCRIPTION
**Pull request by greens**
…n/closed can bring device online

### CODE REVIEW

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy [hunk 1]
```diff
@@ -183,5 +183,5 @@ def configure() {
 
 	// temperature minReportTime 30 seconds, maxReportTime 5 min. Reporting interval if no activity
 	// battery minReport 30 seconds, maxReportTime 6 hrs by default
-	return refresh() + zigbee.batteryConfig() + zigbee.temperatureConfig(30, 300) // send refresh cmds as part of config
+	return refresh() + zigbee.iasZoneConfig(60, 300) + zigbee.batteryConfig() + zigbee.temperatureConfig(120, 300) // send refresh cmds as part of config
```
Comment (varzac) [inline devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:186]: The minReportingInterval can stay the same for temp, it's the max that matters.  I would leave the min at 30, but max up to 600.  @tpmanley that work for you?

**Comment by varzac** [comment]
Any reason not to do this as a part of https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/3432/files?  Or is the intent to replace that PR?

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy (hunk 1 above)
Comment (tpmanley) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:186]: I think you could go even higher with the max for temp like 3600 (an hour) seems good to me. It's going to get pushed immediately if it changes and I don't see much benefit of sending it more frequently.

**Comment by greens** [comment]
Just slipped my mind that there was already a PR

**Pull request closed without merging**